rpcserver.go

```diff
@@ -4482,6 +4482,26 @@ func (r *rpcServer) createRPCClosedChannel(
 		channel.Resolutions = append(channel.Resolutions, rpcResolution)
 	}
 
+	if dbChannel.LocalFCInfo != nil {
+		localFCInfo := dbChannel.LocalFCInfo
+		htlcActions := make(map[string]*lnrpc.ListHTLCHash)
+		for action, htlcs := range localFCInfo.HtlcActions {
+			htlcHashes := make([][]byte, len(htlcs))
+			for i, htlc := range htlcs {
+				htlcHashes[i] = htlc.RHash[:]
```

Comment:
We need to use `copy()` here to avoid running into the famous [golang loop variable issues](https://mohamed-abdel-mohaimen.medium.com/using-loop-iterator-variable-issues-in-golang-b76c17fb71b6).